src/utils/cli.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+from argparse import ArgumentParser
+
+from src.utils.constants import LogLevels, RunLevels
+
+
+class CLI:
+    """Opens the program with specific networking and logging settings."""
+
+    run_level = None
+    log_level = None
+    log_file = ""
+
+    def setup(self, args):
+
+        parser = ArgumentParser(
+            prog="python<x> main.py",
+            description="Bellboy embedded Python 3 program, for use on Rasperry Pi 3/4 systems.",
+            epilog="To develop locally, use DEBUG and LOCAL_STANDALONE settings.",
+        )
+        log = logging.getLogger("CLI")
+
+        # Log Level
+        parser.add_argument(
+            "-l",
+            dest="log_level",
+            type=str,
+            default=LogLevels.DEBUG.name,
```

Comment:
preferred default to info

Reply:
...and I prefer `debug` as default, mostly because it's extra typing during development.